src/Wizard/FSTokens.ts

```diff
@@ -18,18 +26,33 @@ export class FSTokens implements Tokens {
 
   public readTokens(): Credentials | undefined {
     logger.debug('Reading saved tokens from file %s', this.path);
+    let result: Credentials;
+    for (const path of this.paths) {
+      if (existsSync(path)) {
+        logger.debug('File found. Returns the tokens.');
+        const resultRaw: Buffer = readFileSync(this.path);
+        const fileResult = JSON.parse(
+          resultRaw.toString('utf8')
+        ) as Credentials;
+        result = Object.assign(result ?? {}, fileResult);
+      }
```

Comment:
The logic is unclear. If both files are present, you're using the result of the last reading. Instead, you should return the credentials as soon as they're available.